corehq/apps/style/static/style/less/bootstrap3/report.less

```diff
@@ -0,0 +1,23 @@
+.report-actions {
+  padding: 5px 15px;
+  border-bottom: 1px solid @cc-neutral-hi;
+  text-align: right;
+}
+
+.report-loading-container {
+  position: relative;
+}
+.report-loading {
+  position: absolute;
+  background-color: #ffffff;
+  padding: 15px 20px;
+  z-index: 100;
```

Comment:
how about putting a variable for this in variables.less? why is this 100, what does it need to be above/below?